dynamodb.tf

```diff
@@ -3,6 +3,7 @@ resource "aws_dynamodb_table" "css-dynamodb-table" {
   name         = "${aws_appconfig_application.AppConfigAgentApplication.id}.${var.tables[count.index].name}"
```

Comment:
☝️  
Count up there needs a bump. or `length(var.tables)`? 🤔 
It's just out of reach of a suggestion 😭